packages/web-app-files/tests/unit/components/AppBar/AppBar.spec.ts

```diff
@@ -146,7 +147,8 @@ function getShallowWrapper(
   const mocks = {
     ...defaultComponentMocks({
       currentRoute
-    })
+    }),
+    $archiverService: mock<ArchiverService>()
```

Comment:
Should we just add it to defaultComponentMocks?

Reply:
Hmm, I'm not sure because those services are still located in the files app only. I think I'd prefer them to not sit in our global `defaultComponentMocks`.

Reply:
Fair enough